packages/web-app-files/src/views/spaces/GenericSpace.vue

```diff
@@ -348,6 +353,10 @@ export default defineComponent({
     ]),
 
     async performLoaderTask(sameRoute: boolean, path?: string, fileId?: string | number) {
+      if (this.loadResourcesTask.isRunning) {
```

Comment:
For the record: If `space` and `item` (= relative item path in that space) **both** changed, the loader task is called twice. This is skipping one of the task executions.